### ISSUE TITLE
Description of table names

### ISSUE THREAD

**Issue by brindakv** [informatics-isi-edu/protein-database#14]
Can we provide a short description / comment for table names?

**Comment by svoinea** [comment]
This should be done by setting in the `<table>.py` files the variable `table_comment = <the description you want>`.

**Comment by carlkesselman** [comment]
You can set comments on columns too

Carl

Sent from my iPad

----------------------------------------------------------
Dr. Carl Kesselman
Dean’s Professor, Epstein Department of Industrial and Systems Engineering
Fellow, Information Sciences Institute
Viterbi School of Engineering

Professor, Preventive Medicine
Keck School of Medicine

University of Southern California
4676 Admiralty Way, Suite 1001, Marina del Rey, CA <x-apple-data-detectors://1/0>  90292-6695<x-apple-data-detectors://1/0>
Phone:  +1 (310) 448-9338<tel:+1%20(310)%20448-9338>
Email:  carl@isi.edu<mailto:carl@isi.edu>
Web: http://www.isi.edu/~carl


On Nov 27, 2019, at 2:35 PM, svoinea <notifications@github.com> wrote:

﻿

This should be done by setting in the <table>.py files the variable table_comment = <the description you want>.

**Comment by brindakv** [comment]
I tried adding `table_comment` to `<table>.py`. I get the following error when I run the code: 

>>python3 citation.py --replace comments
Traceback (most recent call last):
  File "citation.py", line 290, in <module>
    main(catalog, mode, replace)
  File "citation.py", line 282, in main
    updater.update_table(mode, schema_name, table_def, replace=replace, really=really)
  File "/usr/local/lib/python3.7/site-packages/deriva/utils/catalog/manage/update_catalog.py", line 118, in update_table
    raise CatalogUpdaterException(msg="Unknown mode {}".format(mode))
deriva.utils.catalog.manage.update_catalog.CatalogUpdaterException

**Comment by carlkesselman** [comment]
This has been fixed.

Carl

----------------------------------------------------------
Dr. Carl Kesselman
Dean’s Professor, Epstein Department of Industrial and Systems Engineering
Fellow, Information Sciences Institute
Viterbi School of Engineering

Professor, Preventive Medicine
Keck School of Medicine

University of Southern California
4676 Admiralty Way, Suite 1001, Marina del Rey, CA  90292-6695
Phone:  +1 (310) 448-9338
Email:  carl@isi.edu<mailto:carl@isi.edu>
Web: http://www.isi.edu/~carl








On Dec 3, 2019, at 12:07 PM, Brinda Vallat <notifications@github.com<mailto:notifications@github.com>> wrote:


I tried adding table_comment to <table>.py. I get the following error when I run the code:

python3 citation.py --replace comments
Traceback (most recent call last):
File "citation.py", line 290, in
main(catalog, mode, replace)
File "citation.py", line 282, in main
updater.update_table(mode, schema_name, table_def, replace=replace, really=really)
File "/usr/local/lib/python3.7/site-packages/deriva/utils/catalog/manage/update_catalog.py", line 118, in update_table
raise CatalogUpdaterException(msg="Unknown mode {}".format(mode))
deriva.utils.catalog.manage.update_catalog.CatalogUpdaterException
